source/common/config/api_type_db.cc

```diff
@@ -0,0 +1,38 @@
+#include "common/config/api_type_db.h"
+
+#include "common/protobuf/protobuf.h"
+
+#include "tools/type_whisperer/api_type_db.pb.h"
+
+namespace Envoy {
+namespace Config {
+
+extern const char* ApiTypeDbPbText;
+
+namespace {
+
+tools::type_whisperer::TypeDb* loadApiTypeDb() {
+  tools::type_whisperer::TypeDb* api_type_db = new tools::type_whisperer::TypeDb;
+  if (Protobuf::TextFormat::ParseFromString(ApiTypeDbPbText, api_type_db)) {
```

Comment:
This will implicitly construct large (260KB) string, slightly prefer use CodedStream to parse.